internal/controller/stages/verification.go

```diff
@@ -20,7 +20,15 @@ import (
 func (r *reconciler) startVerification(
 	ctx context.Context,
 	stage *kargoapi.Stage,
-) (*kargoapi.VerificationInfo, error) {
+) *kargoapi.VerificationInfo {
```

Comment:
Once again, if something goes wrong, we now report it through the VerificationInfo's Phase and Message fields. What we don't want a verification problem to look like a _Stage_ problem.